src/features/pages/p2p/hero/content.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { StaticImage } from 'gatsby-plugin-image'
+import Flex from 'features/components/atoms/flex-box'
+import Typography from 'features/components/atoms/typography'
+import { get_lang_direction, Localize } from 'components/localization'
+import useBreakpoints from 'components/hooks/use-breakpoints'
+
+const DP2HeroContent = () => {
+    const { is_mobile } = useBreakpoints()
+    return (
+        <Flex.Box
+            justify="start"
+            direction="col"
+            align="start"
+            gap="4x"
+            basis="6-12"
+            dir={get_lang_direction()}
+        >
+            <div className="visible-phone-and-tablet">
+                <StaticImage
+                    src="../../../../images/common/p2p/p2p_logo.png"
+                    alt="p2p logo"
+                    width={125}
+                    height={32}
+                    layout="fixed"
+                />
+            </div>
```

Comment:
Did you try to use `FlexBox.Item basis=''` here? I know that sometimes we have to provide specific UI
Just asking

Reply:
No, I didn't try that.